nncf/quantization/algorithms/accuracy_control/rank_functions.py

```diff
@@ -9,26 +9,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Dict
+from typing import Callable, List, TypeVar
 
 import numpy as np
 
+from nncf.common.utils.backend import BackendType
 
-def normalized_mse(x_ref: Dict[str, np.ndarray], x_approx: Dict[str, np.ndarray]) -> float:
+TTensor = TypeVar("TTensor")
+
+
+def create_normalized_mse_func(backend: BackendType) -> Callable[[List[TTensor], List[TTensor]], float]:
+    """
+    Factory method to create backend-specific implementation of the normalized_nmse.
+
+    :param backend: A backend type.
+    :return: The backend-specific implementation of the normalized_nmse.
+    """
+    if backend in (BackendType.ONNX, BackendType.OPENVINO):
```

Comment:
Do we have accuracy control method for ONNX?

Reply:
Fixed